hedera-node/test-clients/src/main/resource/testSystemFiles/throttles-dev.json

```diff
@@ -69,7 +69,8 @@
 			"TokenFreezeAccount",
 			"TokenUnfreezeAccount",
 			"TokenGrantKycToAccount",
-			"TokenRevokeKycFromAccount"
+			"TokenRevokeKycFromAccount", 
+      "TokenClaimAirdrop"
```

Comment:
Could you please adjust the indentation here?

Reply:
Not sure what's going on but in Github it looks weird but when I open it in IntelliJ it looks fine 🤷

Reply:
Is uses spaces instead of tabs

Reply:
Now it should be ok 😄